charts/unified-chart/templates/high-priority/hp-deployment.yaml

```diff
@@ -21,16 +21,16 @@ spec:
     metadata:
       labels:
         {{- include "unified.hp.labels" . | nindent 8 }}
-        {{- with .Values.podLabels  }}
-        {{- toYaml . | nindent 8 }}
-        {{- end }}
       annotations:
         checksum/config: {{ (print .Values.configmap) | sha256sum }}
         {{- with .Values.web.podAnnotations }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
-      {{- with .Values.highPriority.serviceAccountName }}
+      {{- if .Values.serviceAccount.enabled }}
+      serviceAccountName: {{ include "unified.name" . }}
```

Comment:
Please let's use explicit function here, i worry that a change to the unified.name will kill us in the future